examples/homepage.ipynb

```diff
@@ -160,6 +160,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "hvPlots will show widgets like the \"Time\" slider here whenever your data is indexed by dimensions that are not mapped onto the plot axes, allowing you to explore complex datasets much more easily than with the default plotting support.\n",
```

Comment:
Would end this sentence with `allowing you to explore complex datasets much more easily.` (not need for the rest imho).